### PR TITLE
feat: new homepage header

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#1196

image

- update home page header
- fix issue with header logo on md breakpoint

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/components/atoms/RegistryIconLink.tsx [hunk 1]
```diff
@@ -9,8 +9,8 @@ export const RegistryIconLink: React.FC<{ color: string }> = ({ color }) => {
     <Link to="/">
       <Box
         sx={{
-          width: { xs: 100, sm: 117 },
-          height: { xs: 'auto', sm: 77 },
+          width: { xs: 62, md: 117 },
+          height: { xs: 'auto', md: 77 },
```
Comment (flagrede) [inline web-registry/src/components/atoms/RegistryIconLink.tsx:13]: Fix header logo issue on md breakpoint:
image

File: web-registry/src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -59,9 +56,14 @@ const Home: React.FC = () => {
   return (
     <Box sx={{ backgroundColor: 'primary.main' }}>
       <BackgroundImgSection
-        img={heroSection?.background?.image?.asset?.url || ''}
-        linearGradient="linear-gradient(180deg, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 100%);"
+        img={
+          'https://cdn.sanity.io/images/jm12rn9t/production/ae7630b9aa71e3d701b692ed576ba0fbd4c0433a-1440x760.jpg?w=2000&fit=max&auto=format' ||
```
Comment (flagrede) [inline web-registry/src/pages/Home/Home.tsx:60]: The sanity URL is temporary to be able to QA the new design on the preview env. Publishing on sanity will apparently publish in production too so we'll need to merge this PR before publishing sanity new changes.

File: web-registry/src/pages/Home/Home.tsx [hunk 3]
```diff
@@ -99,13 +108,17 @@ const Home: React.FC = () => {
...
-              maxWidth: { xs: '187px', sm: '100%' },
+              maxWidth: ['252px', '560px'],
             }}
           >
             <img
               loading="lazy"
               style={{ width: '100%' }}
-              src={heroSection?.icon?.image?.asset?.url || ''}
+              src={
+                'https://cdn.sanity.io/images/jm12rn9t/production/5402a5e5868601a1920b270178dfbe0b60da6683-590x503.png?w=2000&fit=max&auto=format' ||
```
Comment (flagrede) [inline web-registry/src/pages/Home/Home.tsx:118]: Same here.

File: web-registry/src/pages/Home/Home.tsx [hunk 4]
```diff
@@ -42,8 +41,6 @@ const Home: React.FC = () => {
 
   const { batchesWithSupply, setPaginationParams } = usePaginatedBatches();
 
-  const isMobile = useMediaQuery(theme.breakpoints.down('md'));
```
Comment (flagrede) [inline web-registry/src/pages/Home/Home.tsx:45]: Avoid using client-side responsive check here because it's not necessary.

**Comment by flagrede** [comment]
@erikalogie you can check the new header here: https://deploy-preview-1295--regen-registry.netlify.app/

**Comment by erikalogie** [comment]
This looks great. Just one small alignment thing for desktop: https://monosnap.com/file/yLHadR4eCiW2uAg9QUKPUA6YFWFs66.

Also, I'm wondering what this will look like in the CMS. Will we be able to easily control which of the h1 text is white vs. the gradient? I'm thinking that we will probably need to change the text for launch as we don't have NCTs yet, but I'm not really sure to what, so I just want to make sure it is easily changable in the CMS.

**Comment by flagrede** [comment]
> This looks great. Just one small alignment thing for desktop: https://monosnap.com/file/yLHadR4eCiW2uAg9QUKPUA6YFWFs66.
> 
> Also, I'm wondering what this will look like in the CMS. Will we be able to easily control which of the h1 text is white vs. the gradient? I'm thinking that we will probably need to change the text for launch as we don't have NCTs yet, but I'm not really sure to what, so I just want to make sure it is easily changable in the CMS.

@erikalogie regarding the CMS, unfortunately I had to hard-code the text to handle the gradients. With the current setup it's not possible to add custom style like this while keeping the text wired to the CMS. It would be a small PR to change the title though.

**Review by victordelval — APPROVED** [review]

**Comment by erikalogie** [comment]
@flagrede ok, I thought this was possible to do because we have something similar in the Regen website homepage that works via CMS: https://monosnap.com/file/nMvg3Ahb7d7pzvYxCSuaPajXBnrCHc. But it's not a problem to change this via  small pull request. I've asked Sarah B what the new title should be.

**Comment by flagrede** [comment]
> @flagrede ok, I thought this was possible to do because we have something similar in the Regen website homepage that works via CMS: https://monosnap.com/file/nMvg3Ahb7d7pzvYxCSuaPajXBnrCHc. But it's not a problem to change this via small pull request. I've asked Sarah B what the new title should be.

This case with a link would work indeed because it's possible to define that in the cms. What's not currently possible is defining custom style in the CMS.

**Review by wgwz — APPROVED** [review]

**Review by mhagel — APPROVED** [review]

**Comment by flagrede** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**